vnext/src/Libraries/Components/Pressable/Pressable.windows.js

```diff
@@ -188,6 +189,8 @@ type Props = $ReadOnly<{|
    * Duration to wait after press down before calling `onPressIn`.
    */
   unstable_pressDelay?: ?number,
+
+  tabIndex?: ?number, // [Windows]
```

Comment:
This change is confined to the type, so I wouldn't expect a runtime behavior change. Does tabIndex work differently than before with this change?